test/xds/xds_server_serving_mode_test.go

```diff
@@ -206,13 +214,20 @@ func (s) TestServerSideXDS_ServingModeChanges(t *testing.T) {
 		}
 	})
 
-	// Initialize an xDS-enabled gRPC server and register the stubServer on it.
+	// Initialize an xDS-enabled gRPC server and use the helper to start the test service.
```

Comment:
s/Initialize an xDS-enabled gRPC server and use the helper to start the test service/Initialize an xDS-enabled gRPC server and sets it on stub server.

Reply:
Updated the comments and pushed the changes.